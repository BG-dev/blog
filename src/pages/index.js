import React, {Component} from "react"
import { graphql, Link } from "gatsby"
import '../styles/normalize.css'
import Header from '../components/Header'
import {ContainerStyle, AuthorBlogCardStyle, AvatarStyle, TitleBlogCardStyle, 
  ListBlogCardsStyle, BlogCardStyle, ImageBlogCardStyle, InformationBlogCardStyle} from "../styles/styles"

export class BlogCard extends Component {

  constructor(props){
    super(props);
    this.node = props.data.node;
    this.title = this.node.title;
    this.authorImage = this.node.authorAvatarImage.gatsbyImageData.images.fallback.src;
    this.postImage = this.node.postImage.gatsbyImageData.images.fallback.src;
    this.authorName = this.node.authorName;
    this.creationDate = this.node.creationDate;
  }

  render(){
    return (
      <BlogCardStyle>
        <InformationBlogCardStyle>
          <AuthorBlogCardStyle>
            <AvatarStyle src={this.authorImage} alt="avatar"/>
            <p className="authorName">{this.authorName}</p>
          </AuthorBlogCardStyle>
          <Link to={this.title}><TitleBlogCardStyle>{this.title}</TitleBlogCardStyle></Link>
          <p>{this.creationDate}</p>
        </InformationBlogCardStyle>
        <Link to={this.title}>
          <ImageBlogCardStyle src={this.postImage} alt="post"/>
        </Link>
      </BlogCardStyle>
    )
  }
}


export default class IndexPage extends Component {
  constructor(props) {
    super(props);
    this.data = props.data;   
  }

  render(){
    return(
      <div>
        <Header/>
        <ContainerStyle>
          <ListBlogCardsStyle>
            {this.data.allContentfulBlog.edges.map((edge, index) => <BlogCard key={index}  data={edge}/>)}
          </ListBlogCardsStyle>
        </ContainerStyle>
      </div>
    );
  }
}

export const pageQuery = graphql`
  query pageQuery {
    allContentfulBlog{
      edges {
        node {
          title
            content {
                content
            }
            authorAvatarImage{
                gatsbyImageData
            }
            postImage{
              gatsbyImageData
            }
            authorName
            creationDate
        }
      }
    }
  }
`