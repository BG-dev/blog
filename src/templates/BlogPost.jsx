import { graphql } from "gatsby";
import React from "react";
import PropTypes from 'prop-types'
import Header from '../components/Header'
import {AvatarStyle, ContainerStyle, PostInformationStyle, PostStyle, PostContentStyle} from "../styles/styles"
import { Component } from "react";

class BlogPost extends Component{

    constructor(props){
        super(props)

        this.data = props.data;
        this.contentful = this.data.contentfulBlog;
        this.title = this.contentful.title
        this.content = this.contentful.content.content
        this.image = this.contentful.authorAvatarImage.gatsbyImageData.images.fallback.src;
        this.authorName = this.contentful.authorName;
        this.creationDate = this.contentful.creationDate;
    }

    render(){
        return(
            <div>
                <Header/>
                <ContainerStyle>
                    <PostStyle>
                        <h2>{this.title}</h2>
                        <PostInformationStyle>
                            <AvatarStyle src={this.image} alt="avatar"/>
                            <p className="authorName">{this.authorName}</p>
                            <p>{this.creationDate}</p>
                        </PostInformationStyle>
                        <PostContentStyle>
                            <p>{this.content}</p>
                        </PostContentStyle>
                    </PostStyle>
                </ContainerStyle>
            </div>
        );
    }
}

export default BlogPost

export const pageQuery = graphql`
    query blogPostQuery($title: String!){
        contentfulBlog(title: {eq: $title}) {
            title
            content {
                content
            }
            authorAvatarImage{
                gatsbyImageData
            }
            postImage{
                gatsbyImageData
            }
            authorName
            creationDate
        }
    }
`