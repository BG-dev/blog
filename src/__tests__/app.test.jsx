import React from "react"
import BlogPost from "../templates/BlogPost"
import {BlogCard} from "../pages/index"
import { render } from "@testing-library/react"



describe("Blog Post", () => {
  it("Renders in layout", () => {
    const postData = {
      
      contentfulBlog: {
        authorAvatarImage: {
          gatsbyImageData: {
            images: {
              fallback: {
                src: "https://images.ctfassets.net/qaewhz92nurf/7ITV96IbYOyroFdtTQNxlK/ba144cf418eaf777c6b48ae7de750443/Mikita.Azizau.png?w=1550&h=1465&fl=progressive&q=50&fm=jpg"
              }
            }
          }
        },
        content: {
          content: "Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк."
        },
        postImage: {
          gatsbyImageData: {
            images: {
              fallback: {
                src: "https://images.ctfassets.net/qaewhz92nurf/41RWKhKy3mLqDKugWydnlR/018054cd5e01df4b8a9d2d0627657ae5/pedestrian-road-street-car-new-york-manhattan-taxi-vehicle-nyc-america-contrast-yellow-sports-car-yellow-cab-supercar-street.jpg?w=1152&h=864&fl=progressive&q=50&fm=jpg"
              }
            }
          }
        },
        authorName: "Nikita Azizov",
        creationDate: "2021-12-23",
        title: "Жизнь в Минске"
    }
  }
    const container = render(<BlogPost data={postData} />)
    expect(container).toMatchSnapshot()
  })
})

describe("Blog Card", () => {
  it("Renders in layout", () => {
    const cardData = {
      node:{
        title: "Жизнь в Минске",
        authorAvatarImage: {
          gatsbyImageData: {
            images: {
              fallback: {
                src: "https://images.ctfassets.net/qaewhz92nurf/7ITV96IbYOyroFdtTQNxlK/ba144cf418eaf777c6b48ae7de750443/Mikita.Azizau.png?w=1550&h=1465&fl=progressive&q=50&fm=jpg"
              }
            }
          }
        },
        content: {
          content: "Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк."
        },
        postImage: {
          gatsbyImageData: {
            images: {
              fallback: {
                src: "https://images.ctfassets.net/qaewhz92nurf/41RWKhKy3mLqDKugWydnlR/018054cd5e01df4b8a9d2d0627657ae5/pedestrian-road-street-car-new-york-manhattan-taxi-vehicle-nyc-america-contrast-yellow-sports-car-yellow-cab-supercar-street.jpg?w=1152&h=864&fl=progressive&q=50&fm=jpg"
              }
            }
          }
        },
        authorName: "Nikita Azizov",
        creationDate: "2021-12-23"
      }
  }
    const container = render(<BlogCard data={cardData} />)
    expect(container).toMatchSnapshot()
  })
})