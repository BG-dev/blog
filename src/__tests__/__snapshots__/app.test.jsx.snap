// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Card Renders in layout 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <li
        class="css-1b865cr"
      >
        <div
          class="css-bp38ad"
        >
          <div
            class="css-qr5w71"
          >
            <img
              alt="avatar"
              class="css-1cb8wce"
              src="https://images.ctfassets.net/qaewhz92nurf/7ITV96IbYOyroFdtTQNxlK/ba144cf418eaf777c6b48ae7de750443/Mikita.Azizau.png?w=1550&h=1465&fl=progressive&q=50&fm=jpg"
            />
            <p
              class="authorName"
            >
              Nikita Azizov
            </p>
          </div>
          <a
            href="Жизнь в Минске"
          >
            <h2
              class="css-1q2o24w"
            >
              Жизнь в Минске
            </h2>
          </a>
          <p>
            2021-12-23
          </p>
        </div>
        <a
          href="Жизнь в Минске"
        >
          <img
            alt="post"
            class="css-1hw3njn"
            src="https://images.ctfassets.net/qaewhz92nurf/41RWKhKy3mLqDKugWydnlR/018054cd5e01df4b8a9d2d0627657ae5/pedestrian-road-street-car-new-york-manhattan-taxi-vehicle-nyc-america-contrast-yellow-sports-car-yellow-cab-supercar-street.jpg?w=1152&h=864&fl=progressive&q=50&fm=jpg"
          />
        </a>
      </li>
    </div>
  </body>,
  "container": <div>
    <li
      class="css-1b865cr"
    >
      <div
        class="css-bp38ad"
      >
        <div
          class="css-qr5w71"
        >
          <img
            alt="avatar"
            class="css-1cb8wce"
            src="https://images.ctfassets.net/qaewhz92nurf/7ITV96IbYOyroFdtTQNxlK/ba144cf418eaf777c6b48ae7de750443/Mikita.Azizau.png?w=1550&h=1465&fl=progressive&q=50&fm=jpg"
          />
          <p
            class="authorName"
          >
            Nikita Azizov
          </p>
        </div>
        <a
          href="Жизнь в Минске"
        >
          <h2
            class="css-1q2o24w"
          >
            Жизнь в Минске
          </h2>
        </a>
        <p>
          2021-12-23
        </p>
      </div>
      <a
        href="Жизнь в Минске"
      >
        <img
          alt="post"
          class="css-1hw3njn"
          src="https://images.ctfassets.net/qaewhz92nurf/41RWKhKy3mLqDKugWydnlR/018054cd5e01df4b8a9d2d0627657ae5/pedestrian-road-street-car-new-york-manhattan-taxi-vehicle-nyc-america-contrast-yellow-sports-car-yellow-cab-supercar-street.jpg?w=1152&h=864&fl=progressive&q=50&fm=jpg"
        />
      </a>
    </li>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Blog Post Renders in layout 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <header
          class="css-1hvipe"
        >
          <div
            class="css-dzbrss"
          >
            <a
              href="/"
            >
              <h1>
                Blog
              </h1>
            </a>
          </div>
        </header>
        <div
          class="css-dzbrss"
        >
          <div
            class="css-1t2stdg"
          >
            <h2>
              Жизнь в Минске
            </h2>
            <div
              class="css-29ta8s"
            >
              <img
                alt="avatar"
                class="css-1cb8wce"
                src="https://images.ctfassets.net/qaewhz92nurf/7ITV96IbYOyroFdtTQNxlK/ba144cf418eaf777c6b48ae7de750443/Mikita.Azizau.png?w=1550&h=1465&fl=progressive&q=50&fm=jpg"
              />
              <p
                class="authorName"
              >
                Nikita Azizov
              </p>
              <p>
                2021-12-23
              </p>
            </div>
            <div
              class="css-11ama30"
            >
              <p>
                Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <header
        class="css-1hvipe"
      >
        <div
          class="css-dzbrss"
        >
          <a
            href="/"
          >
            <h1>
              Blog
            </h1>
          </a>
        </div>
      </header>
      <div
        class="css-dzbrss"
      >
        <div
          class="css-1t2stdg"
        >
          <h2>
            Жизнь в Минске
          </h2>
          <div
            class="css-29ta8s"
          >
            <img
              alt="avatar"
              class="css-1cb8wce"
              src="https://images.ctfassets.net/qaewhz92nurf/7ITV96IbYOyroFdtTQNxlK/ba144cf418eaf777c6b48ae7de750443/Mikita.Azizau.png?w=1550&h=1465&fl=progressive&q=50&fm=jpg"
            />
            <p
              class="authorName"
            >
              Nikita Azizov
            </p>
            <p>
              2021-12-23
            </p>
          </div>
          <div
            class="css-11ama30"
          >
            <p>
              Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк. Всю сознательную жизнь мой уровень английского колебался вокруг отметки Intermediate, а представления о Нью-Йорке ограничивались нарезкой сцен из фильма «Манхэттен» Вуди Аллена и сериала «Секс в большом городе». Но однажды я за компанию с друзьями приняла участие в лотерее Green Card. 31 октября 2018 года я буду отмечать двухлетие своего переезда в Нью-Йорк.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
